src/content/docs/en/reference/cli-reference.mdx

```diff
@@ -200,7 +200,15 @@ Runs Astro's development server. This is a local HTTP server that doesn't bundle
 
 Builds your site for deployment. By default, this will generate static files and place them in a `dist/` directory. If [SSR is enabled](/en/guides/on-demand-rendering/), this will generate the necessary server files to serve your site.
 
-Can be combined with the [common flags](#common-flags) documented below.
+### Flags
+
+The command accepts [common flags](#common-flags) and the following additional flags:
```

Comment:
Just a comment, but maybe at some point we stop splitting out common from all? Doesn't have to be in this PR of course!

Reply:
Yeah I think that would be best. I notice our command flags don't overlap a lot actually, except like `--help`, `--verbose`, etc

Reply:
If you'd *like* to do that here, feel free! Otherwise, I'll tackle this at some point separately!

Reply:
I think it might be best to tackle that separately 😅

Reply:
Sure thing! Let's tackle the entire list separately!